content/docs/command-reference/queue/index.md

```diff
@@ -0,0 +1,39 @@
+# queue
+
+A set of commands to manage the
+[DVC experiments](/doc/user-guide/experiment-management/experiments-overview)
+task queue: [start](/doc/command-reference/queue/start),
```

Comment:
I'm not sure about this terminology: "experiment task queue"

Why not just "experiment queue" ? Are "tasks" important enough to differentiate from experiments? (At first glance it sounds like an implementation detail)

Reply:
I think it's important to differentiate between "experiment" vs "queue task" due to commands where there's overlap (like `exp remove` vs `queue remove`).

`exp remove` removes _DVC experiment data_, meaning experiment git refs and their associated DVC cache data. `queue remove` removes _task queue entry data_, meaning the queue entry itself, and associated queue-related artifacts (i.e. logs)

The current plan is also to emphasize this in other commands. `exp show` will be modified so that by default it does not display queued or failed experiments by default (so the default view only shows things that can be removed with `exp remove`). There will still be optional flags for displaying queued/failed exps in the table (since it is useful to be able to see the params/deps for them).

Reply:
@jorgeorpinel Maintaining two separate concepts is important, but feel free to suggest more useful terminology 🙏 .

Reply:
OK I see there's some rationale and justification for the special terminology. I still think "task" is an implementation detail and could be avoided...

> feel free to suggest more useful terminology

I'm thinking that it should be clear that `exp` commands deal with experiment data while `queue` commands deal with experiment queue entries (I mean that was the whole point of separating the commands I think 🙂) so term "experiment" can be used interchangeably in many places. Where disambiguation or emphasis are needed, we can use descriptive wording like "entry from the queue", "worker logs", etc.

But it's something we can follow-up with later if needed (still unsure about the release timeline we're looking at here).

Reply:
Everyone raises good points here. Agreed that `dvc queue` should be used infrequently. No functionality was removed from `dvc exp` here. For example, you can remove queued experiments without using `dvc queue`:

```
$ dvc exp run --queue -S train.min_split=0.5
Queued experiment 'd522711' for future execution.
$ dvc queue status
Task     Name    Created    Status
d522711          02:11 PM   Queued

Worker status: 0 active, 0 idle
$ dvc exp show
 ──────────────────────────────────────────────────────────────────────────────────────>
  Experiment              Created        State    avg_prec   roc_auc   prepare.split   >
 ──────────────────────────────────────────────────────────────────────────────────────>
  workspace               -              -           0.925   0.94602   0.2             >
  10-bigrams-experiment   May 19, 2022   -           0.925   0.94602   0.2             >
  └── d522711             02:11 PM       Queued          -         -   0.2             >
 ──────────────────────────────────────────────────────────────────────────────────────>
$ dvc exp remove d522711
Removed experiments: d522711
```

In this case, `dvc exp remove` and `dvc queue remove` are aliases, but `dvc queue remove` is needed to do things like drop successful experiments from `dvc queue status` and `logs` (and it seemed better to allow flexibility to remove any task from the queue as long as we have the command).

_When the `dvc queue` commands are needed_, it often is helpful to distinguish between the experiment and the queue task. Examples include completed experiments where you might want to clear them and their logs from the queue but keep the results, or checkpoints where there are multiple experiment rows for a single queue task (note that you can use the finished experiment names in the `dvc queue` commands, like `dvc queue logs exp-7002e`).

Ideas to further improve:
* If there's functionality missing from `dvc exp` regarding the queue, we should add it where possible (for example, I notice `dvc exp remove` doesn't work for failed experiments, so I'll open an issue).
* We might want to drop the suggested deprecations here (like dropping `dvc exp remove --queued`).

Reply:
My only specific suggestion for now is to avoid term "task". Be descriptive instead e.g. "queued experiment", "experiment from queue", even "entry from queue" if needed.

Reply:
@jorgeorpinel if `queue` is considered a low level building block (and eventually can do things besides experiments for example), it's fine to use `task` here to my mind.

Reply:
I'm not strongly opposed but it seems unnecessary to me: the whole task management aspect of this is an implementation detail (ultimately irrelevant for users).

Reply:
> queue is considered a low level building block

But "queue" is the name of the command. Hard to avoid that term.

p.s. term "worker process" also seems redundant and too deep. We already have "jobs" for the option name, I'd stick to that.
Combined we have "task queue worker process" -- words have lost all meaning 😋